src/box/lua/init.c

```diff
@@ -159,6 +159,7 @@ extern char session_lua[],
 	config_applier_mkdir_lua[],
 	config_applier_roles_lua[],
 	config_applier_sharding_lua[],
+	config_descriptions_lua[],
 	config_cluster_config_lua[],
```

Comment:
Nit: I suppose, they're sorted alphabetically:
```suggestion
	config_cluster_config_lua[],
	config_descriptions_lua[],
```

Reply:
Done.